controllers/propagator/policy_controller.go

```diff
@@ -41,7 +41,7 @@ var log = ctrl.Log.WithName(ControllerName)
 //+kubebuilder:rbac:groups=apps.open-cluster-management.io,resources=placementrules,verbs=get;list;watch
 //+kubebuilder:rbac:groups=core,resources=events,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=core,resources=configmaps,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=core,resources=secrets,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=*,resources=*,verbs=get;list;watch
```

Comment:
A nice reminder that I still have "better configurability of policy controller roles" on my wishlist... Christmas is coming, I guess ;)

(I think this is fine, for now, but if I was a security-conscious user looking at a tool to manage my k8s clusters' security, I think I would have a company policy against wildcards like this in my roles.)